lnrpc/lightning.proto

```diff
@@ -1809,6 +1809,11 @@ message GetInfoResponse {
     announcements and invoices.
     */
     map<uint32, Feature> features = 19;
+
+    /*
+    Indicates whether the HTLC interceptor API is in always-on mode.
+    */
+    bool require_interceptor = 21;
```

Comment:
Since there are multiple interceptors (RPC middleware interceptor, channel acceptor), rename this to `require_htlc_interceptor`?

Reply:
Good point, fixed